common/constants.go

```diff
@@ -324,3 +325,40 @@ const (
 	ExternalCommandResponseTopicPrefixKey = "CommandResponseTopicPrefix"
 	ResponseTopicPrefixKey                = "ResponseTopicPrefix"
 )
+
+// MessageBus Topics
+const (
+
+	// Common Topics
+	DefaultBaseTopic        = "edgex"         // Used if the base topic is not specified in MessageBus configuration
+	EventsPublishTopic      = "events"        // <ServiceType>/<DeviceServiceName>/<ProfileName>/<DeviceName>/<SourceName> are appended
```

Comment:
In some scenario it might not  be  `<DeviceServiceName>` , see https://github.com/edgexfoundry/edgex-go/pull/4345#issuecomment-1427587507

Reply:
Nope, it needs to be the DeviceSerivceName that the Profile and device go with. See my replay to the referenced comment appove.